src/app/modules/yourself/pages/yourself/yourself.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnDestroy } from '@angular/core';
+import { CurrentContentService } from 'src/app/shared/services/current-content.service';
+
+const YourselfBreadcrumbCat = 'Yourself';
+
+@Component({
+  selector: 'alg-yourself',
+  templateUrl: './yourself.component.html',
+  styleUrls: ['./yourself.component.scss'],
+})
+export class YourselfComponent implements OnDestroy {
```

Comment:
Call it "CurrentUserComponent". "Yourself" is just the current name of the link.

Reply:
Done